schemas/json/widget/widget.schema.v1.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$id": "https://altinncdn.no/schemas/json/widget/widget.schema.v1.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Altinn Studio widget",
+    "description": "Schema that describes a widget for use in Altinn Studio",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+      "components": {
+        "$ref": "#/definitions/components"
+      },
+      "displayName": {
+        "$ref": "#/definitions/displayName"
+      },
+      "texts": {
+        "$ref": "#/definitions/texts"
+      }
+    },
+    "required": ["components", "displayName"],
+    "definitions": {
+      "components": {
+        "title": "The layout components associated with the widget",
+        "description": "Contains data describing the components that are part of the widget",
+        "type": "array",
+        "items": {
+          "$ref": "https://altinncdn.no/schemas/json/layout/layout.schema.v1.json#/definitions/component"
+        }
+      },
+      "displayName": {
+        "title": "The widget disply name",
+        "description": "The text that is shown in the component menu in Altinn Studio",
+        "type": "string",
+        "default": "Widget"
+      },
+      "texts": {
+        "title": "Texts associated with the widget",
+        "description": "Contains an array of text resources that are associates with the widget. Each item in the array represents one language",
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/textResources"
+        }
+      },
+      "textResources": {
+        "title": "Text resources",
+        "description": "A collection of text resources for a given language",
+        "type": "object",
+        "properties": {
+          "language": {
+            "$ref": "#/definitions/language"
+          },
+          "resources": {
+            "$ref": "#/definitions/resources"
+          }
+        },
+        "required": ["language", "resources"]
+      },
+      "language": {
+        "title": "The language",
+        "description": "Defines the language for the text resources",
+        "type": "string",
+        "maxLength": 2
+      },
+      "resources": {
```

Comment:
Got a feeling that this should be called texts and the top one textResources or something... Naming is difficult :)

Reply:
Hehe yes :) Again this is from the actual text resources files and their corresponding C# models, so renaming this would be a big job.